docs/developer-docs/latest/developer-resources/database-apis-reference/rest/relations.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Reordering relations
+description: Use the REST API to manage the order of relations
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/rest/relations-reordering.html
+---
+
+<style lang="scss" scoped>
+@media (min-width: 1536px) {
+  .custom-block.api-call > .response {
+    flex: 0 0 32%;
+  }
+}
+</style>
+
+# Managing relations through the REST API <BetaBadge />
+
+Relations between content-types can be managed through the [admin panel](/user-docs/latest/content-manager/managing-relational-fields.md#managing-multiple-choices-relational-fields) or through [REST](/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md) requests sent to the Content API.
+
+Relations can be added, updated, or removed through the Content API by passing parameters in the body of the request:
+
+|  Parameter name | Description | Type of update |
+|-----------------|------------------|----------------|
+| [`connect`](#connect)       | Adds relation(s).<br /><br />Can be used in combination with `disconnect`.<br /><br />Can be used with [positional arguments](#relations-reordering) to define an order for relations.    | Partial
+| [`disconnect`](#disconnect)    | Removes relation(s).<br /><br />Can be used in combination with `connect`. | Partial
+| [`set`](#set)           | Sets relations, replacing all the existing ones.<br /><br />Must be used alone.  | Full
```

Comment:
```suggestion
| [`set`](#set)           | Set entities to a specific set. Using `set` will overwrite all existing connections to other entities.<br /><br />It can not be used in combination with `connect` or `disconnect`.  | Full
```